### PR TITLE
Brandon allison

### PR DESCRIPTION
**Pull request by BrandonJAllison**

### CODE REVIEW

File: src/05_lists.py [hunk 1]
```diff
@@ -8,22 +8,26 @@
...
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE 
-print(x)
+print(x.extend(y))
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
 # YOUR CODE HERE 
-print(x)
+print(x.pop(4 & 5))
```
Comment (lappjeff) [inline src/05_lists.py]: I've never seen `pop` with multiple indexes passed to it before like `(4 & 5)`, but it seems to work! Alternatives that may be more readable would be `x.remove(8)` to search and remove the passed number(8) or `x.pop(4)`

File: src/05_lists.py [hunk 2]
```diff
@@ -8,22 +8,26 @@
...
 
 # Change x so that it is [1, 2, 3, 4, 9, 99, 10]
 # YOUR CODE HERE 
-print(x)
+z = [99, 10]
+x.pop(5)
+print(x.extend(z))
```
Comment (lappjeff) [inline src/05_lists.py]: This could also be done using insert with index and number params, Ex: `x.insert(5, 99)`

File: src/05_lists.py [hunk 3]
```diff
@@ -8,22 +8,26 @@
...
 
 # Print the length of list x
 # YOUR CODE HERE 
-
+print(len(x))
 # Print all the values in x multiplied by 1000
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+for number in x:
```
Comment (lappjeff) [inline src/05_lists.py:32]: love the descriptive `number` argument for your loop here!

File: src/08_comprehensions.py [hunk 4]
```diff
@@ -27,7 +27,8 @@
 a = ["foo", "bar", "baz"]
 
 y = []
-
+for item in a:
```
Comment (lappjeff) [inline src/08_comprehensions.py]: This is a for loop, which is not technically a list comprehension. Could you refactor this to be a list comprehension. If you need any help with it, reach out to me!

**Review by lappjeff — CHANGES_REQUESTED** [review]

File: src/08_comprehensions.py (hunk 4 above)
Comment (BrandonJAllison) [reply src/08_comprehensions.py]: Should be fixed

File: src/05_lists.py (hunk 1 above)
Comment (BrandonJAllison) [reply src/05_lists.py]: I will take a look at the other ways to do it

**Pull request merged**